texastribune/the-dp
# Add box-sizing of content-box to .add-on

#### What's this PR do?
Add box-sizing of content-box to .add-on to fix box-sizing of border-box currently making buttons render not as intended.

#### Why are we doing this? How does it help us?
Gets us ready for Higher Ed Explorer pointer.

#### How should this be manually tested?
Run locally, inspect elements, and when you check out .add-on, you should see 'box-sizing: content-box' as a style for it. What I did was then go to http://www.texastribune.org/higher-ed/explore/, inspect elements there, insert 'box-sizing: content-box' to .add-on in dev tools, and visually confirm that adding this 'box-sizing: content-box' style fixes the buttons.

#### What are the relevant tickets?
Email ask from Rodney.

#### Next steps?
Get Chris to transfer over ownership of tx_highered on PyPi.
Bump version and run `make release` on master to push the new version to PyPi.

#### Smells?

#### TODOs:

#### Has the relevant documentation/wiki been updated?

#### Technical debt note
